src/tests/__mocks__/slackMocks.ts

```diff
@@ -10,23 +10,23 @@ export const slackUserData: UsersListResponse =
     ]
   };
 
-export const slackUserDataError = {
+export const slackUserSpecialMock: UsersListResponse ={
```

Comment:
This variable name could be updated to 'slackUserSpecialCharacterMock' for greater clarity. I suspect will need to be updated in multiple places so make sure tests pass before changing it and again after changing it.

Reply:
Done